dev/com.ibm.ws.security.oauth/src/com/ibm/ws/security/oauth20/plugins/OidcBaseClientProvider.java

```diff
@@ -175,15 +183,15 @@ public boolean validateClient(String clientIdentifier, String clientSecret) {
         return result;
     }
 
+    @Override
     public OidcBaseClient update(OidcBaseClient newClient) throws OidcServerException {
         if (tc.isEntryEnabled()) {
             Tr.entry(tc, "update");
         }
         if (tc.isEntryEnabled()) {
             Tr.exit(tc, "update");
         }
-
-        throw new OidcServerException(ERROR_DESCRIPTION_UNIMPLEMENTED, OIDCConstants.ERROR_SERVER_ERROR, HttpServletResponse.SC_METHOD_NOT_ALLOWED);
+        throw new OidcServerException(new BrowserAndServerLogMessage(tc, null, ERROR_DESCRIPTION_UNIMPLEMENTED), OIDCConstants.ERROR_SERVER_ERROR, HttpServletResponse.SC_METHOD_NOT_ALLOWED);
```

Comment:
This is one of the ones we mentioned not necessarily needing to use the `BrowserAndServerLogMessage` class because a translated message isn't actually being used here.